src/components/Form/ValidationElement/ValidationElement.jsx

```diff
@@ -67,26 +68,31 @@ export default class ValidationElement extends React.Component {
   /**
    * Handle the key down event.
    */
-  handleKeyDown (event) {
+  handleKeyDown(event) {
     if (this.props.onKeyDown) {
       this.props.onKeyDown(event)
     }
   }
 
-  flattenObject (obj) {
+  flattenObject(obj) {
     let o = flattenObject(obj)
     return o
   }
 
-  mergeError (previous, error) {
+  mergeError(previous, error) {
     return mergeError(previous, error)
   }
 
-  triageErrors (section, previous, codes) {
+  triageErrors(section, previous, codes) {
     return triageErrors(section, previous, codes)
   }
 
-  guid () {
-    return newGuid()
+  guid() {
+    // give a fake GUID so the field IDs don't differ between snapshots
+    if (env.IsTest()) {
+      return newMockGuid()
+    } else {
+      return newGuid()
+    }
```

Comment:
Rather than having the conditional, maybe we should consider just using the counter...

Reply:
You mean in lieu of the randomly generated guid? Guess I don't really see why it needs to be completely unique, but I could definitely be missing something.

Reply:
Yeah, would definitely want to have a better understanding of the GUID usage before doing that.